src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/RemoteTagHelperDeltaProvider.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
`RemoteTagHelperDeltaProvider` is the brains of the operation in the ServiceHub process and is responsible for calculating the actual deltas given a "full" set of TagHelpers. Basically it's given the "full" set of TagHelpers and it needs to look at its last returned TagHelper set and determine the delta between the two